<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cal</class>
 <widget class="QDialog" name="cal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>290</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans CJK HK</family>
     <pointsize>30</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>주차 요금 계산</string>
   </property>
  </widget>
  <widget class="QPushButton" name="find_btn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>100</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{
	background-color: rgb(148, 198, 65);
	color: rgb(255, 255, 255);
	border-radius: 5px
}
QPushButton:hover{
	
	background-color: rgb(78, 154, 6);
	border-radius: 5px

}</string>
   </property>
   <property name="text">
    <string>조 회</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="plate_input">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="title_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>차량 번호</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>201</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>핸드폰 번호</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>220</y>
     <width>201</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>입장 날짜</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>260</y>
     <width>201</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>입장 시간</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>300</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>퇴장 날짜</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>340</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>퇴장 시간</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>380</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>결제 금액</string>
   </property>
  </widget>
  <widget class="QPushButton" name="out_btn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>500</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{
	background-color: rgb(148, 198, 65);
	color: rgb(255, 255, 255);
	border-radius: 5px
}
QPushButton:hover{
	
	background-color: rgb(78, 154, 6);
	border-radius: 5px

}</string>
   </property>
   <property name="text">
    <string>나가기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cal_btn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>500</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{
	background-color: rgb(148, 198, 65);
	color: rgb(255, 255, 255);
	border-radius: 5px
}
QPushButton:hover{
	
	background-color: rgb(78, 154, 6);
	border-radius: 5px

}</string>
   </property>
   <property name="text">
    <string>계 산</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>201</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>보유 포인트</string>
   </property>
  </widget>
  <widget class="QPushButton" name="point_btn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>420</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{
	background-color: rgb(148, 198, 65);
	color: rgb(255, 255, 255);
	border-radius: 5px
}
QPushButton:hover{
	
	background-color: rgb(78, 154, 6);
	border-radius: 5px

}</string>
   </property>
   <property name="text">
    <string>포인트 사용</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="point_input">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>420</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="phone_input">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>460</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="take_btn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>460</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{
	background-color: rgb(148, 198, 65);
	color: rgb(255, 255, 255);
	border-radius: 5px
}
QPushButton:hover{
	
	background-color: rgb(78, 154, 6);
	border-radius: 5px

}</string>
   </property>
   <property name="text">
    <string>포인트 적립</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>460</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>핸드폰 번호</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
